components/skus/common/features.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_SKUS_COMMON_FEATURES_H_
+#define BRAVE_COMPONENTS_SKUS_COMMON_FEATURES_H_
+
+namespace base {
+struct Feature;
+}  // namespace base
+
+namespace skus {
+namespace features {
+
+extern const base::Feature kSkusFeature;
```

Comment:
comments

Reply:
Added comments in https://github.com/brave/brave-core/pull/10847/commits/f1fd81200a1006b5c0971b6805968acf4934c79a